### PR TITLE
fix: copy changes and job list edits

### PR DESCRIPTION
**Pull request by MaxFrank13**
This will make changes to the form prompt text in the Skills Builder. Additionally, the career list has been modified to only include options that have skills associated with them.

The data file has been moved and is now a `json` file rather than being an export from a JS file. This should make it easier to read through for any subsequent changes. Also the `JobTitle` key is now `jobTitle` because seeing the capital letter was breaking my brain — all categories are now lowercase as well (i.e. "ProductManagement" is now "productManagement").

### CODE REVIEW

File: src/skills-builder/skills-builder-steps/data/jobsByCategory.json [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+         "jobTitle": "Coding Instructor"
+      },
+      {
+         "jobTitle": "Cybersecurity Analyst"
+      },
+      {
+         "jobTitle": "DevOps Engineer"
+      },
+      {
+         "jobTitle": "Digital Marketing Consultant"
```
Comment (cdeery) [inline src/skills-builder/skills-builder-steps/data/jobsByCategory.json:25]: Digital Marketing Consultant was tagged as not having skills

File: src/skills-builder/skills-builder-steps/select-preferences/CareerInterestCategorizinator.jsx [hunk 2]
```diff
@@ -9,7 +9,7 @@ import CareerInterestCard from './CareerInterestCard';
 import { addCareerInterest, clearAllCareerInterests, setExpandedList } from '../../skills-builder-context/data/actions';
 import { SkillsBuilderContext } from '../../skills-builder-context';
 import messages from './messages';
-import { careerList } from '../../utils/jobsByCategory';
+import careerList from '../data/jobsByCategory.json';
```
Comment (cdeery) [inline src/skills-builder/skills-builder-steps/select-preferences/CareerInterestCategorizinator.jsx:12]: Nice. I was getting all sorts of warnings about having to assert it as JSON when I tried to do that.

File: src/skills-builder/skills-builder-steps/view-results/ViewResults.jsx [hunk 3]
```diff
@@ -147,16 +147,6 @@ const ViewResults = () => {
       </Row>
     ) : (
       <Stack gap={4.5}>
-        { showMatchesFoundAlert && (
```
Comment (cdeery) [inline src/skills-builder/skills-builder-steps/view-results/ViewResults.jsx:150]: Nice to see that go :-)

File: src/skills-builder/skills-builder-steps/view-results/test/ViewResults.test.jsx [hunk 4]
```diff
@@ -54,7 +54,7 @@ describe('view-results', () => {
       renderSkillsBuilderWrapper();
       // Click the next button to trigger "fetching" the data
       await act(async () => {
-        fireEvent.click(screen.getByRole('button', { name: 'Next Step' }));
+        fireEvent.click(screen.getByRole('button', { name: 'Show Recommendations' }));
```
Comment (cdeery) [inline src/skills-builder/skills-builder-steps/view-results/test/ViewResults.test.jsx:57]: I will usually pull the string I'm looking for from the messages so I don't have to change the tests if the text changes. No biggie, but should we trend toward that in the future?

**Review by cdeery — COMMENTED** [review]
The only actionable thing is the data change. Otherwise it looks good!

File: src/skills-builder/skills-builder-steps/view-results/test/ViewResults.test.jsx (hunk 4 above)
Comment (MaxFrank13) [reply src/skills-builder/skills-builder-steps/view-results/test/ViewResults.test.jsx]: This is a great point! I went ahead and did that for all of the tests as it was pretty trivial to setup a `messages` object in the `setupSkillsBuilder` file. This way all messages are imported in one spot and can be accessed from any test.

**Review by cdeery — APPROVED** [review]
Looks good!

**Pull request merged**